src/KeyRequestHandler.js

```diff
@@ -30,6 +34,11 @@ export default class KeyRequestHandler {
     }
 
     handleKeyRequest(keyRequest) {
+        // Ignore own device key requests if cross-signing lab enabled
```

Comment:
The whole flow to verify one of your own devices with cross-signig happens elsewhere, right?

Reply:
Yes, either via a toast or the right panel.